lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'text'
+require_relative 'menu'
+
+require 'rubygems'
+require 'twilio-ruby'
+
+class Takeaway
+
+include Text, Order, Menu
```

Comment:
These should be objects that `Takeaway` collaborates with, not objects it inherits behaviour from.